### ISSUE TITLE
Codeblocks doesn't scale properly with font size

### ISSUE THREAD

**Issue by wI2L** [squidfunk/mkdocs-material#2027]
- [x ] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [ x] ... the problem doesn't occur with the default MkDocs template
- [ x] ... the problem is not in any of my customizations (CSS, JS, template)
- [ x] ... the documentation does not mention anything about my problem
- [ x] ... there are no open or closed issues that are related to my problem

## Description

In order to reduce the overall size of the markdown content, the following CSS rule was added in an `extra.css` file:
```css
.md-typeset {
    font-size: 0.7rem;
}
```

### Expected behavior

The content of the page should scale accordingly.

### Actual behavior

It appears that code blocks line numbers divs have their own size and line-height, which produce the following result:
<img width="125" alt="68747470733a2f2f66696c65732e6769747465722e696d2f3538663239373432643733343038636534663538613866642f6d6358612f7468756d622f53637265656e2d53686f742d323032302d31312d30332d61742d31322e35372e32312e706e67" src="https://user-images.githubusercontent.com/6519569/97990084-b23a2680-1ddf-11eb-8963-f5983591a139.png">

### Steps to reproduce the bug

1. Edit `mkdocs.yml` and add an extra CSS file:
```yaml
extra_css:
  - stylesheets/extra.css
```
2. Edit the `extra.css` file and change the `font-size` of class `.md-typeset`

### Package versions

* Python: 3.9
* MkDocs: 1.1.2
* Material: 6.1.2

### Project configuration

``` yaml
# elided for brevity
extra_css:
  - stylesheets/extra.css
  - stylesheets/fonts.css
  - stylesheets/admonitions.css
```

### System information

* OS: OSX
* Browser: Firefox

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by wI2L** [comment]
According to @squidfunk on Gitter:
> `.codehilitetable .linenos` defines padding as `rem`. Should be `em`.

**Comment by wI2L** [comment]
@squidfunk 
I have been able to fix the issue temporarily with the following:
```css
.md-typeset {
    font-size: 0.7rem;
}
.codehilitetable .linenos,
.highlighttable .linenos {
    padding: .525em 1.1764705882em !important;
    padding-right: 0 !important;
}
```

**Comment by squidfunk** [comment]
Fixed in aaa65f4d.

**Comment by wI2L** [comment]
👍 Thanks @squidfunk for the quick fix.
